config/jobs/cert-manager/cert-manager-periodics.yaml

```diff
@@ -286,9 +286,9 @@ periodics:
       env:
       - name: K8S_VERSION
         value: "1.19"
-      # Enable CertificateSigningRequest e2e tests for v1.19+ clusters
+      # Enable CertificateSigningRequest and Gateway API e2e tests for v1.19+ clusters
       - name: FEATURE_GATES
-        value: "ExperimentalCertificateSigningRequestControllers=true"
+        value: "ExperimentalCertificateSigningRequestControllers=true\\,ExperimentalGatewayAPISupport=true"
```

Comment:
Why do we need double backslash?

Reply:
As helm interprets a comma as a list of values rather than a single string containing a comma, it needs to be passed to helm as `\,`. In YAML to get a literal backslash you should use `\\`